app/dashboard/page.tsx

```diff
@@ -40,17 +30,28 @@ export default async function Dashboard() {
     redirect('/login');
   }
   
+  // TODO(SWE-67): Grab posts from following users
+  // Sends a request to load the initial posts
+  const payload = { "userIds": ['dtran', 'jfales', 'sfales'] };
```

Comment:
I thought userIds were supposed to be either (1) hash values or (2) graph node ids? I understand the current values are hardcoded, but aren't these example userIds actually usernames?

Reply:
These will be implemented in @sammyoto's PR `[SWE-63]`